fastq/tenXGenomics.go

```diff
@@ -93,12 +84,12 @@ func fastqStats(fq *Fastq) string {
 	return fmt.Sprintf("%s\t%d\n%s\n", fq.Name, len(fq.Seq), dna.BasesToString(fq.Seq))
```

Comment:
Just wondering, is there a reason we tab between the name in length versus putting it on a new line? Is there a goal maybe to keep each entry only two lines? And I assume the new line at the end of this is to aid in human legibility downstream?

Reply:
Seems like fastq format is hardcoded to read and expect the seq on the second line of the record (with name being the first line) to change that would probably be a problematic task and that would only be useful for TenX reads, this way the name can be the first and the seq the second and you can store the length as well (that what it seems like is going on here). The way we have every file type coded they all need to end in a new line. If there are more fastqs to put in the file, it will write to this new line created in the record before it, but at the end of the file there should be an empty new line, and every reader we've built (at least that i've encountered or written) will fail if the file doesnt end in a new line character.